static/src/stylesheets/layout/garnett-header/_menu.scss

```diff
@@ -0,0 +1,116 @@
+.menu {
+    background-color: $nav-background-colour;
+    box-sizing: border-box;
+    font-size: 20px;
+    left: 0;
+    line-height: 1;
+    margin-right: $gutter-small + ($veggie-burger-medium / 2);
+    padding-bottom: $gs-baseline * 2;
+    top: 0;
+    z-index: $zindex-main-menu;
+
+    .new-header--slim & {
+        margin-right: $gs-gutter / 2 + ($veggie-burger-large / 2);
+    }
+
+    @include mq($until: desktop) {
+        @include menu-animation(-110%);
+        -webkit-overflow-scrolling: touch;
+        box-shadow: 3px 0 16px rgba(0, 0, 0, .4);
+        bottom: 0;
+        height: 100%;
+        overflow: auto;
+        padding-top: $gs-baseline / 2;
+        position: fixed;
+        right: 0;
+        transition: transform .2s cubic-bezier(.23, 1, .32, 1);
+        will-change: transform;
+    }
+
+    @include mq(mobileMedium) {
+        margin-right: $gutter-small + ($veggie-burger-medium / 2);
+    }
+
+    @include mq(mobileLandscape) {
+        margin-right: $gutter-medium + ($veggie-burger-medium / 2);
+    }
+
+    @include mq(desktop) {
+        display: none;
+        position: absolute;
+        padding-bottom: 0;
+        padding-top: 0;
+        top: 100%;
+        // Left, right and width values are fallback for if vw is not supported
+        left: 0;
+        right: 0;
+        width: 100%;
+        border-bottom: 1px solid $rule;
+
+        @supports (width: 100vw) {
+            // based on https://css-tricks.com/full-width-containers-limited-width-parents/#article-header-id-6
+            left: 50%;
+            right: 50%;
+            width: 100vw;
+            margin-left: -50vw;
+            margin-right: -50vw;
+        }
+
+        .new-header--slim & {
+            top: $slim-nav-height;
+        }
+    }
+
+    .new-header--open & {
+        @include mq($until: desktop) {
+            @include menu-animation(0%);
+        }
+
+        // TODO: replace with sexy animation
```

Comment:
Is this todo still important?

Reply:
If we decide to experiment with animation the TODO won't have been the reason : ) I will get rid.